bpf/lib/lb.h

```diff
@@ -1627,6 +1709,13 @@ static __always_inline int lb4_local(const void *map, struct __ctx_buff *ctx,
 	if (lb4_svc_is_affinity(svc))
 		lb4_update_affinity_by_addr(svc, &client_id, backend_id);
 #endif
+
+#if defined(ENABLE_LOCAL_REDIRECT_POLICY) && defined(HAVE_NETNS_COOKIE)
+	if (netns_cookie > 0 && unlikely(lb4_svc_is_localredirect(svc)) &&
+	    lb4_skip_xlate_from_ctx_to_svc(netns_cookie, tuple->daddr, tuple->sport))
+		return CTX_ACT_OK;
+#endif /* ENABLE_LOCAL_REDIRECT_POLICY && HAVE_NETNS_COOKIE */
```

Comment:
I meant moving this logic for the case `saddr == backend->address`, so that the map lookups are done only for loopback case.

Reply:
Done